solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/LTRFeatureLoggerTransformerFactory.java

```diff
@@ -61,7 +62,8 @@ public DocTransformer create(String name, SolrParams params,
 
     // Hint to enable feature vector cache since we are requesting features
     req.getContext().put(CommonLTRParams.LOG_FEATURES_QUERY_PARAM, true);
-    req.getContext().put(CommonLTRParams.STORE, params.get(CommonLTRParams.STORE));
+    req.getContext().put(CommonLTRParams.FV_STORE, params.get(CommonLTRParams.FV_STORE));
+    req.getContext().put(CommonLTRParams.FV_FORMAT, params.get(CommonLTRParams.FV_FORMAT));
```

Comment:
I don' think it does.  FV_RESPONSE_WRITER uses the get with a default param. I wanted to follow the same format that it was doing. It will also accept null and default to sparse later in the code. All the bases should be covered